mitodl/mitxonline
# Adds logic to avoid stepping on an in-progress basket when processing checkout responses

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #453 

#### What's this PR do?
Fixes #453 

#### How should this be manually tested?
Create a basket, convert to an Order, and then set the is_review flag to True on the resultant order. Then, create a new basket and attempt to process the previous Order. The basket should remain untouched. (Not sure how you'd get CyberSource to put the transaction into a review state but you can do that too if possible.) 

You should also be able to create a basket, go through the checkout process, hit the payment page, then go back (in a separate window), create a new basket again, and then complete the payment process for the first basket. The cart in the separate window should not be affected (you will have to reload the page). Checking out that cart normally should still clear the cart.
